SteamBus.App/src/Steam.Session/SteamSession.cs

```diff
@@ -856,6 +854,7 @@ or EResult.Expired
 
   public void SaveToken()
   {
+    Console.WriteLine("#### TEST");
```

Comment:
Is this test output still needed? If not, should it be removed or the text changed?

Reply:
Removed it, sorry about that